R/controls.R

```diff
@@ -30,7 +30,7 @@ jqueryLib <- function() {
 }
 
 #' @export
-filter_select <- function(id, label, sharedData, group, allLevels = FALSE,
+filter_select <- function(id, label, sharedData, group = "default", allLevels = FALSE,
   multiple = TRUE) {
```

Comment:
This is a different kind of group--it's a group in the dplyr sense. The select box lets you choose subsets of the rows to display; the group argument indicates how those subsets should be determined. I know this is confusing, we can pick different/better names.